src/app/main/PluginGrid.tsx

```diff
@@ -107,21 +110,27 @@ export class PluginGrid extends React.Component<IPluginGridProps, IState> {
    */
   private loadPlugins() {
     // splits out install required and !install required plugins
-    const plugins = getPlugins()
+    const plugins = getPlugins().filter(p => typeof (p as any).component !== 'undefined')
     const noInstallNeeded = plugins.filter(p => !p.requiresInstall) as IInstalledPlugin[]
     const installNeeded = plugins.filter(p => p.requiresInstall) as IInstallNeededPlugin[]
     
+    log.info(`starting load for [${noInstallNeeded.map(e => e.name).join(',')}]`)
+
     // load installed plugins
     this.setState({
       plugins: noInstallNeeded
     })
 
+    log.info(`starting install for [${installNeeded.map(e => e.name).join(',')}]`)
+
     // install the uninstalled plugins
     Promise.all(this.installPlugins(installNeeded)).then((installedPlugins) => {
       // when these load, merge them in
       this.setState({
         plugins: this.state.plugins.concat(installedPlugins)
       })
+    }, (error) => {
+      log.error(`failed installation: ${error}`)
```

Comment:
if `error` is an object it may be printed like so: `failed installation: [object Object]`
if it is an instance of `Error` then it will print like so: `failed installation: Error: <message>`

I recommend using `error.message` when logging errors through template strings

Reply:
would prefer to refactor template content to expect Error: message than rely on a specific object shape here. will make the change - good catch!

Reply:
this looks like the only outstanding comment then its good to merge ✨